code/modules/telesci/telepad.dm

```diff
@@ -51,18 +51,33 @@
 
 	return ..()
 
+var/global/list/cargopads = list() // Global List of Cargo Pads
 
 //CARGO TELEPAD//
 /obj/machinery/telepad_cargo

```

Comment:
Create a new telepad object and put it in the hippiestation module, please leave the current code unchanged for the sake of easier upstream merges